client/src/app.jsx

```diff
@@ -25,11 +25,11 @@ class App extends React.Component {
       loading: true,
     }
     this.getGame = this.getGame.bind(this);
-    this.url = 'http://localhost:3001/game/'
+    this.url = 'http://localhost:3001/api/'
   }
 
   componentDidMount() {
-    const gid = window.location.pathname.match(/\/games\/(\d+)\//);
+    const gid = window.location.pathname.match(/\/cartgames\/(\d+)\//);
```

Comment:
I think this one might not have been necessary, and might even wind up being undesirable (I think eventually we're going to have to find a way to coordinate endpoints so that everyone's bundle knows what data to fetch). But for now I don't think it'll cause any problems.